.circleci/scripts/setup_ci_environment.sh

```diff
@@ -2,7 +2,7 @@
 set -ex -o pipefail
 
 # Set up NVIDIA docker repo
-if [[ "${BUILD_ENVIRONMENT}" == *cuda* ]]; then
+if [[ "${BUILD_ENVIRONMENT}" == *cuda* ]] || [[ "${BUILD_ENVIRONMENT}" == *cu* ]]; then
```

Comment:
`*cu*` should cover both cases, shouldn't it?
```suggestion
if [[ "${BUILD_ENVIRONMENT}" == *cu* ]]; then
```

Reply:
Changed to this, it'd probably good for us to look to actually just get rid of the `BUILD_ENVIRONMENT` variable, I feel like it causes more trouble then benefits